lib/kb_quast/authclient.py

```diff
@@ -44,8 +44,8 @@ def add_valid_token(self, token, user):
         with self._lock:
             self._cache[token] = [user, _time.time()]
             if len(self._cache) > self._maxsize:
-                for i, (t, _) in enumerate(sorted(self._cache.items(),
-                                                  key=lambda (_, v): v[1])):
+                for i, (t, _) in enumerate(sorted(list(self._cache.items()),
```

Comment:
Looks like this client is out of date compared to the other auth clients

Reply:
It is. If you don't generate the python client, these files don't get updated. I opted just to remove them (and change the test to use the auth client in installed_clients)